F21/bhargavareddy24/BRCodeReview-01.py

```diff
@@ -0,0 +1,41 @@
+class WordLowerCase:
+    def init(self, word, language):
+        self.word = word
+        self.l = language
+
+    def lower(self):

```

Comment:
I like that you kept all the functions contained within a class that handled the replacement/lowercase in one function